test/testdata/infer/star_star_args.rb

```diff
@@ -38,12 +38,12 @@ def main
         one_kwarg(foo: "bad", a: "bad") # error: Expected `Integer` but found `String("bad")` for argument `foo`
         one_kwarg(foo: 1, a: 1) # error: Expected `String` but found `Integer(1)` for argument `args`
         one_kwarg(foo: "bad", a: 1) # error: Expected `Integer` but found `String("bad")` for argument `foo`
-      # ^^^^^^^^^^^^^^^^^^^^^^^^^^^ error: Expected `String` but found `Integer(1)` for argument `args`
+      #           ^^^^^^^^^^^^^^^^ error: Expected `String` but found `Integer(1)` for argument `args`
```

Comment:
I think if you convert some of the errors in this file from `# error:` to `# ^^^ error:` it'll be more clear that all keyword args share a single loc.

Reply:
Happy to be proven wrong with a test though!

Reply:
This fixes kwargs; I forgot to add my new test.

Reply:
I think the issue in this specific file is `**` parameters.